src/utils/connectors.ts

```diff
@@ -0,0 +1,30 @@
+import { InjectedConnector } from '@web3-react/injected-connector'
+import { WalletConnectConnector } from '@web3-react/walletconnect-connector'
+import { WalletLinkConnector } from '@web3-react/walletlink-connector'
+
+const NETWORK_URL = process.env.REACT_APP_ETHEREUM_RPC_URL
+
+export const NETWORK_CHAIN_ID: number = parseInt(process.env.REACT_APP_ETHEREUM_NETWORK_ID ?? '1')
```

Comment:
It's a [nullish coalescing operator](javascripttutorial.net/es-next/javascript-nullish-coalescing-operator/#:~:text=ES2020%20introduced%20the%20nullish%20coalescing,double%20question%20marks%20(%20%3F%3F%20).&text=It%20returns%20the%20right%20operand,is%20either%20null%20or%20undefined%20.
). It uses the first value if it's not `null` or `undefined`, or uses the 2nd if it is `null` or `undefined`. The reason it exists instead of using the `||` operator is that `||` can incorrectly pass through falsey values like `''` and `0` on the first value.